WholeGenomeGermlineSingleSample.wdl

```diff
@@ -104,6 +108,22 @@ workflow WholeGenomeSingleSampleQc {
       preemptible_tries = papi_settings.agg_preemptible_tries
   }
 
+  # Estimate level of cross-sample contamination
+   call Processing.CheckContamination as CheckContamination {
+     input:
+       input_bam = input_bam,
+       input_bam_index = input_bam_index,
+       contamination_sites_ud = contamination_sites_ud,
+       contamination_sites_bed = contamination_sites_bed,
+       contamination_sites_mu = contamination_sites_mu,
+       ref_fasta = ref_fasta,
+       ref_fasta_index = ref_fasta_index,
+       output_prefix = sample_name + ".verify_bam_id",
+       preemptible_tries = papi_settings.agg_preemptible_tries,
+       contamination_underestimation_factor = 0.75
```

Comment:
Is it fine to leave this hardcoded or should it be an optional input with a default or something?

Reply:
new issue created for follow up